Tasks/SSH/ssh.ts

```diff
@@ -94,6 +95,18 @@ async function run() {
                 tl.debug('Copying script to remote machine.');
                 await sshHelper.copyScriptToRemoteMachine(scriptFile, scpConfig);
 
+                //change the line encodings
+                var isWin = os.type().match(/^Win/);
+                var windowsEncodedRemoteScriptPath = remoteScriptPath;
+                if (isWin) {
+                    remoteScriptPath = remoteScriptPath + ".unix";
```

Comment:
Maybe change `.unix` to `._unix` to avoid the .0001% chance that we conflict with an existing file?